ci/docker/centos-7-cpp.dockerfile

```diff
@@ -30,7 +30,7 @@ RUN yum install -y \
 
 ARG cmake=3.23.1
 RUN wget -nv -O - https://github.com/Kitware/CMake/releases/download/v${cmake}/cmake-${cmake}-Linux-x86_64.tar.gz | tar -xzf - -C /opt

```

Comment:
@assignUser could you perhaps update the install setup to place it under `/opt/cmake-${cmake}` instead, so the path is always correct regardless the cmake version?

Reply:
Thanks for the suggestion, tested it successfully here: https://github.com/assignUser/test-repo-a/runs/6458346977?check_suite_focus=true#step:6:392